db-tools-ij/src/main/resources/com/splicemachine/db/loc/toolsmessages_de_DE.properties

```diff
@@ -86,8 +86,7 @@ IJ_Fata01=Schwerwiegende Ausnahme\: {0}\: {1}
 -- f\u00FChrt die Anweisung mit Parameterwerten\n\                               -- von der USING-ResultSet-Zeile durch\n\  REMOVE Name;                 -- entfernt die benannte zuvor vorbereitete Anweisung\n\ \n\  RUN ''Dateiname'';              -- f\u00FChrt Befehle von der benannten Datei aus\n\ \n\  ELAPSEDTIME [ ON | OFF ];    -- legt den Modus f\u00FCr abgelaufene Zeit f\u00FCr ij fest\n\  MAXIMUMDISPLAYWIDTH Ganzzahlwert;\n\                               -- legt die maximale Anzeigebreite f\u00FCr\n\                               -- jede Spalte auf einen Ganzzahlwert fest\n\ \n\  ASYNC Name ''SQL-J-Text'';     -- f\u00FChrt den Befehl in einem anderen Thread aus\n\  WAIT FOR Name;               -- warten auf Ergebnis von Befehl f\u00FCr ASYNC\n\ \n\  GET [SCROLL INSENSITIVE] [WITH  '{HOLD | NOHOLD}'] CURSOR Name AS ''SQL-J-Abfrage'';\n\                               -- ruft einen Cursor (JDBC-ResultSet) f\u00FCr die Abfrage ab\n\                               -- der Standard ist ein Forward-Only-Cursor mit Holdability\n\  NEXT Name;                   -- ruft die n\u00E4chste Zeile aus dem benannten Cursor ab\n\  FIRST Name;                  -- ruft die erste Zeile aus dem benannten Scroll-Cursor ab\n\  LAST Name;                   -- ruft die letzte Zeile aus dem benannten Scroll-Cursor ab\n\  PREVIOUS Name;               -- ruft die vorige Zeile aus dem benannten Scroll-Cursor ab\n\  ABSOLUTE Ganzzahlname;       -- positioniert den benannten Scroll-Cursor an der absoluten Zeilenzahl\n\                               -- (Eine negative Zahl gibt die Position von der letzten Zeile an.) \n\  RELATIVE ganzeZahl Name;        -- platziert den benannten Scroll-Cursor relativ zur aktuellen Zeile\n\       -- (ganzeZahl ist die Anzahl der Zeilen)\n\  AFTER LAST Name;                -- platziert den benannten Scroll-Cursor hinter der letzten Zeile\n\  BEFORE FIRST Name;              -- platziert den benannten Scroll-Cursor vor der ersten Zeile\n\  GETCURRENTROWNUMBER Name;       -- gibt f\u00FCr die aktuelle Position des benannten Scroll-Cursors die Zeilennummer an\n\                                  -- (Befindet sich der Cursor nicht in einer Zeile, wird 0 zur\u00FCckgegeben.) \n\  CLOSE Name;                     -- schlie\u00DFt den benannten Cursor\n\  LOCALIZEDDISPLAY [ ON | OFF ];\n\  -- steuert die Darstellung von Daten, die von der L\u00E4ndereinstellung abh\u00E4ngen\n\ \n\  EXIT;                                             -- beendet IJ\n\  HELP;                 -- zeigt diese Nachricht an\n\\n\  SPOOL '<DATEI>';                 -- schreibt SQL Befehle und ihre Ergebnisse in <DATEI>\n\  SPOOL STOP;                 -- h\u00F6rt auf, SQL Befehle und ihre Ergebnisse in Datei zu schreiben\n\  SPOOL CLEAR;                 -- l\u00F6scht Daten aus der Datei, in die SQL Befehle und ihre Ergebnisse geschrieben sind\n\ \n\ Nicht erkannte Befehle werden als potenzielle SQL-J-Befehle angesehen und direkt ausgef\u00FChrt.\n\ \nIJ_IjErro0=IJ-FEHLER\: {0}
```

Comment:
This might be a stupid question, but the newly added SPOOL message will show in German if `IJ_IjErro0=IJ-FEHLER\: {0}` is deleted?

Reply:
fixed and cleaned up the German help message.